sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/Quaternion.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.MixedReality.ObjectAnchors.Models
+{
+    /// <summary> The Quaternion. </summary>
+    internal partial class Quaternion : IEquatable<Quaternion>
+    {
+        internal System.Numerics.Quaternion data;
+
+        internal Quaternion(System.Numerics.Quaternion quaternion)
+        {
+            data = quaternion;
+        }
+
+        /// <summary> Initializes a new instance of Quaternion. </summary>
+        /// <param name="x"> . </param>
+        /// <param name="y"> . </param>
+        /// <param name="z"> . </param>
+        /// <param name="w"> . </param>
+        /// <param name="isIdentity"> . </param>
+#pragma warning disable CA1801 // Review unused parameters
+        internal Quaternion(float x, float y, float z, float w, bool isIdentity) : this(x, y, z, w)
+#pragma warning restore CA1801 // Review unused parameters
+        {
+        }
+
+        /// <summary> Initializes a new instance of Quaternion. </summary>
+        /// <param name="x"> . </param>
+        /// <param name="y"> . </param>
+        /// <param name="z"> . </param>
+        /// <param name="w"> . </param>
+        internal Quaternion(float x, float y, float z, float w)
+        {
+            data = new System.Numerics.Quaternion(x, y, z, w);
+        }
+
+        public float X { get { return data.X; } set { data.X = value; } }
```

Comment:
Missing docs.

Reply:
It's an internal class, is this necessary? I think most of the fields are self-explanatory anyway